iterator/git.go

```diff
@@ -239,7 +239,10 @@ func (obj *Git) Recurse(ctx context.Context, scan interfaces.ScanFunc) ([]interf
 
 	// make a unique ID for the directory
 	// XXX: we can consider different algorithms or methods here later...
-	sum := sha256.Sum256([]byte(obj.URL))
+	// Added commit hash due to error when scanning
+	// https://webrtc.googlesource.com/src using different commit hashes and
+	// without different commit hashes.
+	sum := sha256.Sum256([]byte(obj.URL + obj.Hash))
```

Comment:
Very excellent. Actually we want this to be Hash, Rev, or Ref. But there's a catch. How do we _really_ want this to be? Think carefully. Great learning example.

Reply:
Fixed